README.md

```diff
@@ -63,22 +63,20 @@ const actionCreator = () => ({
 });
 ```
 
-It is also possible to use a function:
+If you include [thunk middleware](https://github.com/gaearon/redux-thunk) in your middleware stack, it is also possible to use a function to dispatch multiple actions:
 
 ```js
 const actionCreator = () => ({
   type: 'FIRST_ACTION_TYPE',
   payload: {
-    promise: Promise.resolve((action, dispatch, getState) => {
-      dispatch({ type: 'SECEOND_ACTION_TYPE', payload: ... })
+    promise: Promise.resolve((dispatch, getState) => {
```

Comment:
Waaaait.... I looked at [`index.spec.js`](https://github.com/pburtchaell/redux-promise-middleware/blob/master/test/index.spec.js) now, and see the action there, but I don't really understand what is happening there. And I have to admit that I didn't get it work this way and used `redux-thunk` directly in my use case.

I understand this change should be reverted, but on the other hand I'd like to see an explanation about why you modify redux-thunk. I don't see the convenience you mentioned.